transactions/README.md

```diff
@@ -48,6 +47,14 @@ fileName:transaction.json
       "latitude": 51.5152346,
       "longitude": -0.1310718
     },
+    "offer": {
```

Comment:
I don't think we should put an extended offer on this section. 
Because:
- At the moment only 1 customer uses the offer from the API
- There is a full section explaining in detail the how it looks a "qualified-transaction.json"